metric_source/local/local_test.go

```diff
@@ -60,50 +57,50 @@ func TestEvaluateTarget(t *testing.T) {
 	var until int64 = 67
 	var retention int64 = 10
 	var metricsTTL int64 = 3600
-	metricErr := fmt.Errorf("Ooops, metric error")
+	//metricErr := fmt.Errorf("Ooops, metric error")
 
 	Convey("Errors tests", t, func() {
-		Convey("Error while ParseExpr", func() {
-			dataBase.EXPECT().GetMetricsTTLSeconds().Return(metricsTTL)
-			result, err := localSource.Fetch("", from, until, true)
-			So(err, ShouldResemble, ErrParseExpr{target: "", internalError: parser.ErrMissingExpr})
-			So(err.Error(), ShouldResemble, "failed to parse target '': missing expression")
-			So(result, ShouldBeNil)
-		})
-
-		Convey("Error in fetch data", func() {
-			dataBase.EXPECT().AllowStale().Return(dataBase)
-			dataBase.EXPECT().GetPatternMetrics(pattern).Return([]string{metric}, nil)
-			dataBase.EXPECT().GetMetricRetention(metric).Return(retention, nil)
-			dataBase.EXPECT().GetMetricsValues([]string{metric}, from, until).Return(nil, metricErr)
-			dataBase.EXPECT().GetMetricsTTLSeconds().Return(metricsTTL)
-			result, err := localSource.Fetch("super.puper.pattern", from, until, true)
-			So(err, ShouldResemble, metricErr)
-			So(result, ShouldBeNil)
-		})
-
-		Convey("Error evaluate target", func() {
-			dataBase.EXPECT().AllowStale().Return(dataBase)
-			dataBase.EXPECT().GetPatternMetrics("super.puper.pattern").Return([]string{metric}, nil)
-			dataBase.EXPECT().GetMetricRetention(metric).Return(retention, nil)
-			dataBase.EXPECT().GetMetricsValues([]string{metric}, from, until).Return(dataList, nil)
-			dataBase.EXPECT().GetMetricsTTLSeconds().Return(metricsTTL)
-			result, err := localSource.Fetch("aliasByNoe(super.puper.pattern, 2)", from, until, true)
-			So(err.Error(), ShouldResemble, "Unknown graphite function: \"aliasByNoe\"")
-			So(result, ShouldBeNil)
-		})
-
-		Convey("Panic while evaluate target", func() {
-			dataBase.EXPECT().AllowStale().Return(dataBase)
-			dataBase.EXPECT().GetPatternMetrics("super.puper.pattern").Return([]string{metric}, nil)
-			dataBase.EXPECT().GetMetricRetention(metric).Return(retention, nil)
-			dataBase.EXPECT().GetMetricsValues([]string{metric}, from, until).Return(dataList, nil)
-			dataBase.EXPECT().GetMetricsTTLSeconds().Return(metricsTTL)
-			result, err := localSource.Fetch("movingAverage(super.puper.pattern, -1)", from, until, true)
-			expectedErrSubstring := strings.Split(ErrEvaluateTargetFailedWithPanic{target: "movingAverage(super.puper.pattern, -1)"}.Error(), ":")[0]
-			So(err.Error(), ShouldStartWith, expectedErrSubstring)
-			So(result, ShouldBeNil)
-		})
+		//Convey("Error while ParseExpr", func() {
```

Comment:
What is the reason for commenting this test? I do not see the reasons to skip this test. But even if test is incorrect or is not necessary it should be removed instead of commenting.